tearup.yaml

```diff
@@ -795,7 +795,24 @@
         enabled: yes
         state: restarted
 
-    - name: create the VIP
+    - name: create the VIPs
       shell: |
         #!/bin/bash
-        ip address add {{ vip}} dev lo
+        ip address add {{ item }} dev lo
+        ip rule add from {{ item }} to {{ ansible_facts.default_ipv4.network }}/{{ (ansible_facts.default_ipv4.address + '/' + ansible_facts.default_ipv4.netmask) | ipaddr('prefix') }} lookup 999
```

Comment:
Do we want to make this idempotent?

Reply:
yeah I pushed this only in case my laptop goes on fire. I'll re-work it.